app/Http/Controllers/CelebrityController.php

```diff
@@ -2,9 +2,65 @@
 
 namespace App\Http\Controllers;
 
+use Carbon\Carbon;
+use Illuminate\Support\Facades\DB;
+use App\Models\Celebrity;
 use Illuminate\Http\Request;
 
 class CelebrityController extends Controller
 {
-    //
+    public function index() //print all data in Celebrity database
+    {
+        return view('celebrities.index', [
+            'celebrities' => Celebrity::paginate(5)
```

Comment:
Hey @GuoqingLibro good use of paginate. But I think that we should make this a bit more refined. 

When passing something to a view, we should also consider how do we want our data to be displayed. Like for example, the order of our data.